lib/datasource.js

```diff
@@ -1586,11 +1594,19 @@ DataSource.prototype.discoverSchemas = function(tableName, options, cb) {
     // Default name mapper
     nameMapper = function mapName(type, name) {
       if (type === 'table' || type === 'model') {
-        return fromDBName(name, false);
+        return fromDBName(name, pascalCase);
       } else if (type == 'fk') {
-        return fromDBName(name + 'Rel', true);
+        if (disableCamelCase === true) {
```

Comment:
any reason we cannot use `if(disableCamelCase)`?

Reply:
👍 I thought I need to check the type as well turns out I didn't need to, changed.